stdlib/logging/config.pyi

```diff
@@ -15,9 +16,47 @@ if sys.version_info >= (3, 7):
 else:
     _Path = StrPath
 
+_TKey = TypeVar("_TKey")
+_TValue = TypeVar("_TValue")
+
 DEFAULT_LOGGING_CONFIG_PORT: int
 RESET_ERROR: int  # undocumented
 IDENTIFIER: Pattern[str]  # undocumented
+_listener: Optional[Any]
+dictConfigClass = DictConfigurator
+
+class ConvertingMixin(Generic[_TKey, _TValue]):
+    def convert_with_key(self, key: _TKey, value: _TValue, replace: bool = ...) -> Dict[_TKey, _TValue]: ...
+    def convert(self, value: _TValue) -> Dict[_TKey, _TValue]: ...
+
+class ConvertingDict(ConvertingMixin[_TKey, _TValue]):
+    def __getitem__(self, key: _TKey) -> _TValue: ...
+    def get(self, key: _TKey, default: Optional[_TKey] = ...) -> Union[_TValue, _TKey]: ...
+    def pop(self, key: _TKey, default: Optional[_TKey] = ...) -> Union[_TValue, _TKey]: ...
+
+class ConvertingList(ConvertingMixin[_TKey, _TValue]):
```

Comment:
I think the key type is always `int`, since that's used for `the_list[key]` giving a value:

```suggestion
class ConvertingList(Generic[_TValue], ConvertingMixin[int, _TValue]):
```